src/test/librbd/test_notify.py

```diff
@@ -141,8 +141,10 @@ def slave(ioctx):
         assert(list(image.list_snaps()) == [])
 
         print("rebuild object map")
-        image.update_features(RBD_FEATURE_OBJECT_MAP | RBD_FEATURE_FAST_DIFF,
-                              False)
+        features = image.features() & (
+                RBD_FEATURE_OBJECT_MAP | RBD_FEATURE_FAST_DIFF)
+        if features:
+            image.update_features(features, False)
         image.update_features(RBD_FEATURE_OBJECT_MAP, True)
```

Comment:
Jewel throws an "InvalidArgument" exception if you try to disable a feature that wasn't enabled (or try to enable a feature that was already enabled).